package.yaml

```diff
@@ -28,7 +30,7 @@ dependencies:
   - utf8-string
   - optparse-applicative
 
-ghc-options: -Wall -Werror
+ghc-options: -threaded -rtsopts -Wall -Werror
```

Comment:
Could you explain why these extra options are necessary?

Reply:
I am not an expert in this openssl library but without the `-threaded` option we face
`ConnectionFailure user error (RTS doesn't support multiple OS threads (use ghc -threaded when linking))` at run time when trying to connect to vault. So it seems that openssl requires a threaded runtime.

`-rtsopts` enable all RTS options processing and can be configured at runtime through command line or environment variable. It enables users to configure the number of threads, the heap size, ... We could remove it until we know that we really need it as it might have security implications.